src/App.js

```diff
@@ -2,19 +2,19 @@ import './App.css';
 import {BrowserRouter, Routes, Route} from 'react-router-dom'
 import Nav from './components/nav';
 import ArticleList from './components/articleList';
-import ArticleListByTopic from './components/articleListByTopic';
 import SingleArticle from './components/singleArticle';
 
 function App() {
+
   return (
     <BrowserRouter>
         <div className="App">
           <Nav />
           <Routes>
-            <Route path ="/articles" element={<ArticleList/>}/>
-            <Route path="/cooking" element={<ArticleListByTopic topic={"cooking"}/>}/>
-            <Route path="/coding" element={<ArticleListByTopic topic={"coding"}/>}/>
-            <Route path="/football" element={<ArticleListByTopic topic={"football"}/>}/>
+            <Route path ="/articles" element={<ArticleList topic={""} />}/>
+            <Route path="/cooking" element={<ArticleList topic={"cooking"} />}/>
```

Comment:
Routes for topics must not be hard-coded, please instead think of a way you can fetch the topics from your api and render a list (maybe mapping through the results!)